src/hooks/usePollStakingBalances.ts

```diff
@@ -39,14 +39,14 @@ const usePollStakingBalances = () => {
     try {
       const balancePromises = [];
 
-      // for (const stakingPool of Object.values(StakingPool)) {
-      balancePromises.push(
-        contractClient.stakingPoolClient?.getUserBalancesAndUnclaimedRewards({
-          stakingPool: StakingPool.Liquidity,
-          walletAddress,
-        })
-      );
-      // }
+      for (const stakingPool of Object.values(StakingPool)) {
+        balancePromises.push(
+          contractClient.stakingPoolClient?.getUserBalancesAndUnclaimedRewards({
+            stakingPool,
+            walletAddress,
+          })
+        );
+      }
```

Comment:
```
const balancePromises = Object.values(StakingPool).map(stakingPool => contractClient.stakingPoolClient?.getUserBalancesAndUnclaimedRewards({
            stakingPool,
            walletAddress,
          }))
```

Reply:
Will leave this refactor and the one below to keep these out of the scope of this PR. Will come back and update in a future PR